src/dimension.c

```diff
@@ -381,9 +381,19 @@ ts_dimension_get_open_slice_ordinal(Dimension *dim, DimensionSlice *slice)
 	/* Find the index (ordinal) of the chunk's slice in the open dimension */
 	i = ts_dimension_vec_find_slice_index(vec, slice->fd.id);
 
-	Assert(i >= 0);
-
-	return i;
+	if (i >= 0)
+		return i;
+	else
+	{
+		/*
+		 * Returns the number of slices if the slice not found, i.e., i = -1.
```

Comment:
Better to error out since we don't expect this code path to be reached for regular tables?

Reply:
I don't follow. Returning error means that creating a chunk table will always fail if no space dimension exists.